src/utils/blocked-uas.ts

```diff
@@ -1,22 +1,15 @@
 export const DEFAULT_BLOCKED_UA_STRS = [
-    'ahrefsbot',
-    'ahrefssiteaudit',
+    // Random assortment of bots
     'amazonbot',
     'amazonproductbot',
-    'app.hypefactors.com', // Buck, but "buck" is too short to be safe to block
+    'app.hypefactors.com',
```

Comment:
why remove the comment? Buck is more well-known than this string

Reply:
Oh yuck, removed it by mistake when I was copying stuff around, let me add it back